pkgs/data/misc/libkkc-data/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchurl, pythonPackages }:
+
+stdenv.mkDerivation rec {
+  pname = "libkkc-data";
+  version = "0.2.7";
+  name = "${pname}-${version}";
+  libkkc_version = "0.3.5";
```

Comment:
This attribute can go in a `let ... in` expression

Reply:
Do you mean only `libkkc_version`? If so -- fixed.